package.json

```diff
@@ -21,6 +21,8 @@
     "test": "mocha"
   },
   "dependencies": {
+    "@patch-technology/patch": "^1.18.0",
```

Comment:
Should this say 1.19.0? I edited it directly but it seems like it auto corrected back to 1.18.0

Reply:
i actually dont think we need this dependency here at all? Seems weird that the package depending on an older version of itself

Reply:
I defer to the broader team's judgement here